File: test/aidbox_sdk/snapshots/typescript/hl7-fhir-r4-core/EnrollmentRequest.ts
import { DomainResource } from "./DomainResource";
import { Reference } from "./Reference";
import { Identifier } from "./Identifier";

export type EnrollmentRequest = DomainResource & {
    status?: string;
    created?: string;
    insurer?: Reference;
    coverage?: Reference;
    provider?: Reference;
    candidate?: Reference;
    identifier?: Identifier[];
};

File: test/aidbox_sdk/snapshots/python/hl7-fhir-r4-core/EnrollmentRequest.py
from typing import Optional, List
from base import DomainResource
from base import Reference
from base import Identifier

class EnrollmentRequest(DomainResource):
    status: Optional[str] = None
    created: Optional[str] = None
    insurer: Optional[Reference] = None
    coverage: Optional[Reference] = None
    provider: Optional[Reference] = None
    candidate: Optional[Reference] = None
    identifier: Optional[List[Identifier]] = None